src/utils/currency/tokenPriceDataByName.ts

```diff
@@ -95,14 +96,14 @@ export const fetchTokenPriceDataByName = async ({
         );
       }
 
-      console.error(
-        `Unable to get price for ${tokenName}. It probably doesn't have a listed exchange value.`,
+      debugLogging(
+        `CoinGecko: Unable to get price for ${tokenName}. It probably doesn't have a listed exchange value.`,
       );
       return fallbackResponseData;
     });
   } catch (e) {
     if (!import.meta.env.DEV) {
```

Comment:
I think it no longer makes sense to have this condition given we use `debugLogging`. Even wondering why the condition is different from the previous one 🫠